app/api/items/get_attempts.go

```diff
@@ -39,18 +40,21 @@ type itemAttemptsViewResponseRow struct {
 // swagger:operation GET /items/{item_id}/attempts items itemAttemptsView
 // ---
 // summary: List attempts for a task
-// description: Returns attempts made by the current user (if `items.has_attempts` = 0) or his
-//              teams (if `items.has_attempts` = 1) while solving the task given in `item_id`.
+// description: Returns attempts made by the current user (if `as_team_id` is not given) or
+//              `as_team_id` (otherwise) while solving the task given in `item_id`.
 //
 //
-//              The current user should have at least 'content' permission on the task item,
-//              otherwise an empty list is returned.
+//              The current user (if no `as_team_id` given) or `as_team_id` (otherwise) should have
+//              at least 'content' permission on the task item, the 'forbidden' error is returned.
```

Comment:
.+ the user must be member of the team if `as_team_id` is provided

Reply:
added the note